### ISSUE TITLE
photospline documentation cleanup (Trac #1182)

### ISSUE THREAD

**Issue by icecube-trac** [icecube-trac/tix2#1176]
* add maintainer inforamtion to rst documentaion
* merge information from README into rst documentaion
* although none of the code uses doxygen documentiaon add link to it in rst docs
* add trunk section to RELEASE_NOTES
* The following changes since last release probably should be in the change log: 134517,133997,133772,132130 
*

Migrated from https://code.icecube.wisc.edu/ticket/1182
```json
{
    "status": "closed", 
    "changetime": "2019-02-13T14:11:57", 
    "description": "* add maintainer inforamtion to rst documentaion\n* merge information from README into rst documentaion\n* although none of the code uses doxygen documentiaon add link to it in rst docs\n* add trunk section to RELEASE_NOTES\n* The following changes since last release probably should be in the change log: 134517,133997,133772,132130 \n*", 
    "reporter": "kjmeagher", 
    "cc": "", 
    "resolution": "fixed", 
    "_ts": "1550067117911749", 
    "component": "combo reconstruction", 
    "summary": "photospline documentation cleanup", 
    "priority": "normal", 
    "keywords": "", 
    "time": "2015-08-19T12:15:10", 
    "milestone": "", 
    "owner": "jvansanten", 
    "type": "defect"
}
```

**Comment by icecube-trac** [comment]
Trac update at 20160215T13:38:12: 
* jvansanten commented:

> In changeset 141859:
> 
> ```text
> #!CommitTicketReference repository="IceCube" revision="141859"
> Remove obsolete README and update release notes. Closes #1176.
> ```


* jvansanten changed status from "new" to "closed"
* jvansanten changed resolution from "" to "fixed"

**Comment by icecube-trac** [comment]
Trac update at 20190213T14:11:57: 
* olivas changed milestone from "IceRec V5" to ""
* olivas commented:

> Milestone IceRec V5 deleted
